code/modules/gunse/modular_guns.dm

```diff
@@ -766,10 +766,39 @@ ABSTRACT_TYPE(/obj/item/gun/modular)
 		barrel = new /obj/item/gun_parts/barrel/soviet(src)
 		stock = new /obj/item/gun_parts/stock/italian(src)
 
+/obj/item/gun/modular/soviet/misfitreciver //just a reciver for the guns vendor
+	name = "\improper Misfit"
+	real_name = "\improper Misfit"
+	desc = "A spacer design slapped together from salvaged soviet parts, the Misfit is known for it's ease of manufacture and durablity."
+	max_ammo_capacity = 0 //comes with a mag
+	gun_DRM = GUN_SOVIET
+	spread_angle = 15
+	jam_frequency_fire = 35
```

Comment:
I appreciate this is for zipgunish stuff but wow that's gonna round up to like 40-45% jam rate when you add even the best of parts to it.

Reply:
It also gets a ten gun round mag when it spawns in normally, I'll reduce it though by 10 for the fire frequency (though quick aside I do think it being high adds a bit of flavour to the whole slapped together in someones backyard out of whatever was laying around that day)

Reply:
It also gets a ten gun round mag when it spawns in normally (as in placed on the map), I'll reduce it though by 10 for the fire frequency (though quick aside I do think it being high adds a bit of flavour to the whole slapped together in someones backyard out of whatever was laying around that day)

Reply:
Reduced to 15 on fire, 4 on reload. Jam frequency on fire moved to barrels and reloads moved to magazines. The better stat wise, the more it's gonna jam on you. Probably still needs tweaking in that regard.